apps/api/src/auth/auth.module.ts

```diff
@@ -21,7 +23,17 @@ import { GitlabStrategy } from '@/config/oauth-strategy/gitlab/gitlab.strategy'
         algorithm: 'HS256'
       }
     }),
-    UserModule
+    UserModule,
```

Comment:
Do we really need to define the config in 2 places?

Reply:
Sorry I didn't get your question I removed the config from the global level?

Reply:
Okay great! I didn't notice that